sigar_linux.go

```diff
@@ -793,6 +787,25 @@ func readFile(file string, handler func(string) bool) error {
 	return nil
 }
 
+// Read the first line of a file, ignoring any error
+func readFileLine(file string) string {
+	f, err := os.Open(file)
+	if err != nil {
+		return ""
+	}
+	defer f.Close()
+
+	reader := bufio.NewReader(f)
+	line, _ := reader.ReadString('\n')
+	return strings.TrimSpace(line)
+}
+
+// Convert the value to an uint64, ignoring any error
+func readUint(val string) uint64 {
+	result, _ := strconv.ParseUint(val, 10, 64)
+	return result
```

Comment:
Can you add a test for an unparseable input here? The errors for [ParseUint/ParseInt](https://golang.org/pkg/strconv/#ParseInt) either result in the returned uint being zero, or the maximum value (depending on the error), so I'm thinking you should probably handle the error by setting it to a specific value rather than relying on those default values in the error cases.